crates/biome_analyze/CONTRIBUTING.md

```diff
@@ -318,6 +320,44 @@ impl Rule for ExampleRule {
 }
 ```
 
+#### Navigating the CST
+
+Then navigating the nodes and tokens of certain nodes, you will notice straight away that the majority of those methods will return a `Result` (`SyntaxResult`).
+
+Generally, you will end up navigating the CST inside the `run` function, and this function will usually return an `Option` or a `Vec`.
+
+- If the `run` function returns an `Option`, you're encouraged to transform `Result` into `Option` and use the try operator `?`. This will make your coding way easier:
```

Comment:
```suggestion
- If the `run` function returns an `Option`, you're encouraged to transform the `Result` into an `Option` and use the try operator `?`. This will make your coding way easier:
```